### DISCUSSION TITLE
Table Versioning Metadata

### DISCUSSION THREAD

**Discussion by ConorHuh** [lancedb/lancedb#1089] [Q&A]
In a table's version there are 3 fields: a version, a timestamp and a metadata field. How can I add metadata to these versions to better understand the respective version within a table? For context, here is the schema of the table that I wish to include metadata for when modifying. 

```
schema = pa.schema(
            [
                pa.field(
                    "vector", pa.list_(pa.float32(), list_size=feature_vector_length)
                ),
                pa.field("labels", pa.list_(pa.string(), list_size=-1)),
                pa.field("labels_string", pa.string()),
                pa.field("version", pa.int64())
            ]
        )
```

Thank you

**Comment by wjones127** [comment]
Hi @ConorHuh, while there is indeed a place in the format for this metadata, it is currently not read or written to by our library.

We have an issue upstream tracking work on that. https://github.com/lancedb/lance/issues/514
